.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,69 @@
+<!--
+     For a timely review/response, please avoid force-pushing additional
+     commits if your PR already received reviews or comments.
```

Comment:
I would put this comment later. Not as the first sentence of a PR. If someone hasn't even received any round of reviews, this will be confusing to them